### PR TITLE
Typos and some syntax clarity

### PR DESCRIPTION
**Pull request by paul-f-maxson**
Only changes to text, not to any of the example code

### CODE REVIEW

File: src/configuration/plugin-configuration.md [hunk 1]
```diff
@@ -79,7 +79,7 @@ A common usecase is extending the default config, for this reason the `extends`
 {% endsamplefile %}
 {% endsample %}
 
-To add additional plugins while not overwriting the but default ones, add `"..."` to use the plugins provided by config specified in the `extends` field. For example to add an additional reporter:
+To add additional plugins while not overwriting the default ones, add `"..."` to use the plugins provided by the config specified in the `extends` field. For example to add an additional reporter:
```
Comment (mischnic) [inline src/configuration/plugin-configuration.md]:
```suggestion
To add additional plugins without overwriting the default ones, add `"..."` to use the plugins provided by the config specified in the `extends` field. For example to add another reporter:
```

File: src/configuration/plugin-configuration.md [hunk 2]
```diff
@@ -169,15 +169,15 @@ As you can see, `...` is now used to make sure that `data-url:./worker.js` will
 
 If you're curious how this can be achieved without a deeper integration with Parcel core:
 
-`@parcel/transformer-inline-string` sets marks the asset to be an inlined asset. `@parcel/packager-js` then inlines this inline bundle (as a string `"${contents}"`). This inline bundle was previously processed by `@parcel/optimizer-data-url` which encodes the JS code into a data url.
+`@parcel/transformer-inline-string` marks the asset to be an inlined asset. `@parcel/packager-js` then inlines this inline bundle (as a string `"${contents}"`). This inline bundle was previously processed by `@parcel/optimizer-data-url`, which encodes the JS code into a data url.
```
Comment (mischnic) [inline src/configuration/plugin-configuration.md]:
```suggestion
`@parcel/transformer-inline-string` marks the asset as an inlined asset. `@parcel/packager-js` then inlines this inline bundle (as a string `"${contents}"`). This inline bundle was previously processed by `@parcel/optimizer-data-url`, which encoded the JavaScript code into a data url.
```

**Review by mischnic — APPROVED** [review]
Thanks! That's what I get for not reading that page after writing it 🙈

**Comment by paul-f-maxson** [comment]
> Thanks! That's what I get for not reading that page after writing it 🙈

No problem! I think I tend to read a lot closer than the average person, glad I could contribute my skills to this project!

**Pull request merged**